topics/w4-visual-debugging.md

```diff
@@ -1,11 +1,5 @@
-# W4 - Visual Debugging
-
-### Reading List
-
-#### Required Reading
-* [PythonTutor](https://pythontutor.com/): Guo, P. J. (2013, March). Online python tutor: embeddable web-based program visualization for cs education. In Proceeding of the 44th ACM technical symposium on Computer science education (pp. 579-584).
-
-#### Optional Reading
-* [Ply:](https://github.com/sliminality/ply) Lim, S., Hibschman, J., Zhang, H., & O'Rourke, E. (2018, October). Ply: A visual web inspector for learning from professional webpages. In Proceedings of the 31st Annual ACM Symposium on User Interface Software and Technology (pp. 991-1002).
-* [Log-it](https://creativity.ucsd.edu/logit): Jiang, P., Sun, F., & Xia, H. (2023, April). Log-it: Supporting Programming with Interactive, Contextual, Structured, and Visual Logs. In Proceedings of the 2023 CHI Conference on Human Factors in Computing Systems (pp. 1-16).
-
+# Leveraging Visual Debugging for Programming Education
```

Comment:
The blog post should introduce the general topic of visual debugging before diving into its specific application in programming education.

Consider start with explaining what is visual debugging in both dev and ed contexts.